marimo/_plugins/stateless/mpl/_mpl.py

```diff
@@ -341,3 +281,110 @@ def start_server() -> None:
             height="550px",
         )
     )
+
+
+html_content = """
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="mpl/_static/css/page.css" type="text/css" />
+    <link rel="stylesheet" href="mpl/_static/css/boilerplate.css" type="text/css" />
+    <link rel="stylesheet" href="mpl/_static/css/fbm.css" type="text/css" />
+    <link rel="stylesheet" href="mpl/_static/css/mpl.css" type="text/css" />
+    <script src="mpl/mpl.js"></script>
+    <style>
+    %(custom_css)s
+    </style>
+
+    <script>
+      function ondownload(figure, format) {
+        window.open('download.' + format, '_blank');
+      };
+
+      function ready(fn) {
+        if (document.readyState != "loading") {
+          fn();
+        } else {
+          document.addEventListener("DOMContentLoaded", fn);
+        }
+      }
+
+      ready(
+        function() {
+          var websocket_type = mpl.get_websocket_type();
+          var websocket = new websocket_type("%(ws_uri)sws");
+
+          // mpl.figure creates a new figure on the webpage.
+          var fig = new mpl.figure(
+              // A unique numeric identifier for the figure
+              %(fig_id)s,
+              // A websocket object
+              websocket,
+              // A function called when a file type is selected for download
+              ondownload,
+              // The HTML element in which to place the figure
+              document.getElementById("figure"));
+        }
+      );
+    </script>
+
+    <title>marimo</title>
+  </head>
+
+  <body>
+    <div id="figure"></div>
+  </body>
+</html>
+""".strip()
+
+# Custom CSS to make the mpl toolbar fit the marimo UI
+# We do not support dark mode at the moment as the iframe does not know
+# the theme of the parent page.
+css_content = """
+body {
+    background-color: transparent;
+}
+.ui-dialog-titlebar + div {
+    border-radius: 4px;
+}
+.ui-dialog-titlebar {
+    display: none;
```

Comment:
i removed the title bar - you can customize what it says, but im sure 99% of the time it will say `Figure <random_numbers>`